slack_sdk/socket_mode/aiohttp/__init__.py

```diff
@@ -200,19 +256,25 @@ async def connect(self):
             self.wss_uri = await self.issue_new_wss_url()
         self.current_session = await self.aiohttp_client_session.ws_connect(
             self.wss_uri,
+            autoping=False,
```

Comment:
This is the key change in this pull request. I've decided to stop relying on the AIOHTTP's ping-pong to better track the actual liveness of the connections on our end. 

We've added manual ping-pong communication and `last_ping_pong_time` as the state of the current session. If the value is too old, this Socket Mode client tries to reconnect anyways. This is a proven way in the non-asyncio Socket Mode client. We haven't any similar bug reports for the non-asyncio client.